mitodl/bootcamp-ecommerce
# Update login/registration UI to match designs

Blocked by #521

#### What are the relevant tickets?
Closes #522
Closes #564

#### What's this PR do?
- copies NotificationContainer from xpro, enables it, gets it working
- moves `userNotifications` state inside `ui` state
- some styling and content changes
- fixes the forgot password functionality by adding djoser settings

#### How should this be manually tested?
- Try to hit all the login, registration, and profile pages and see if they look like they're supposed to and still work like they're supposed to.
- Go to `../forgot-password`, submit the form, click the link in the email you should get, and submit that form as well.  Your password should be successfully changed.

#### Any background context you want to provide?
I did not change the default size of the recaptcha to be larger than its native size, but did center it. 

#### Screenshots (if appropriate)

<img width="844" alt="Screen Shot 2020-05-21 at 4 21 25 PM" src="https://user-images.githubusercontent.com/187676/82604474-61f94000-9b82-11ea-8511-fa48a0698c58.png">

-------------

<img width="844" alt="Screen Shot 2020-05-21 at 4 21 45 PM" src="https://user-images.githubusercontent.com/187676/82604494-6b82a800-9b82-11ea-833b-ab17f88e2fa6.png">

-------------


<img width="904" alt="Screen Shot 2020-05-26 at 8 19 17 AM" src="https://user-images.githubusercontent.com/187676/82900125-0df7ae00-9f2a-11ea-9026-9a09b7daec47.png">

-------------

<img width="916" alt="Screen Shot 2020-05-26 at 8 21 05 AM" src="https://user-images.githubusercontent.com/187676/83038236-ba10c600-a00a-11ea-9586-8b87df9a2f43.png">


-------------

<img width="916" alt="Screen Shot 2020-05-26 at 8 21 37 AM" src="https://user-images.githubusercontent.com/187676/82900178-22d44180-9f2a-11ea-8d8e-02de106726b5.png">

-------------


<img width="933" alt="Screen Shot 2020-05-21 at 4 33 15 PM" src="https://user-images.githubusercontent.com/187676/82604424-4db54300-9b82-11ea-8706-673baa0a5f88.png">


-------------

<img width="932" alt="Screen Shot 2020-05-21 at 4 33 04 PM" src="https://user-images.githubusercontent.com/187676/82604435-51e16080-9b82-11ea-8ae5-f9f12d2007c3.png">


--------------

<img width="851" alt="Screen Shot 2020-05-26 at 8 59 38 AM" src="https://user-images.githubusercontent.com/187676/82906964-02a98000-9f34-11ea-8618-8c9defe9037f.png">

----------------

<img width="851" alt="Screen Shot 2020-05-26 at 8 59 51 AM" src="https://user-images.githubusercontent.com/187676/82906983-089f6100-9f34-11ea-9e60-57a6cec5eb41.png">

-----------------

<img width="819" alt="Screen Shot 2020-05-26 at 9 06 55 AM" src="https://user-images.githubusercontent.com/187676/82907004-0f2dd880-9f34-11ea-92e0-16634ec164b0.png">

-----------------

<img width="773" alt="Screen Shot 2020-05-26 at 9 44 47 AM" src="https://user-images.githubusercontent.com/187676/82908210-a9425080-9f35-11ea-9ed1-ace69e12c308.png">
